src/main/java/uk/ac/ebi/eva/pipeline/io/readers/UnwindingItemStreamReader.java

```diff
@@ -0,0 +1,45 @@
+package uk.ac.ebi.eva.pipeline.io.readers;
+
+import org.springframework.batch.item.ExecutionContext;
+import org.springframework.batch.item.ItemReader;
+import org.springframework.batch.item.ItemStreamReader;
+import org.springframework.batch.item.NonTransientResourceException;
+import org.springframework.batch.item.ParseException;
+import org.springframework.batch.item.UnexpectedInputException;
+
+import java.util.Collection;
+import java.util.Iterator;
+
+/**
+ * The unwinding reader takes a reader that returns a collection of elements in each read call and acts as a
+ * buffer that pass single elements in each read call. This class can only be used with ItemStreamReaders
+ *
+ * @param <T>
+ */
+public class UnwindingItemStreamReader<T> extends UnwindingItemReader<T> implements ItemStreamReader<T> {
```

Comment:
Added new test battery... that calls the open method from the unwinder! So now we have tests.